### ISSUE TITLE
Breach & Clear Deadline - CUSA03742

### ISSUE THREAD

**Issue by GAMERRPCS3** [red-prig/fpps4-game-compatibility#1441]
### Game Name

Breach & Clear Deadline

### Game code

CUSA03742

### Game version

01.00

### Used emulator's version

6e5a849

### Current status

Playable

### Error

[main:13672] nop nid:libScePosix:4D4FDDF4F7C81CF3:socket

### Description

starts, and crashes after the intro before getting to the menu.

### Used hacks

DEPTH_DISABLE_HACK, IMAGE_LOAD_HACK, DISABLE_SRGB_HACK

### Screenshots

![1](https://user-images.githubusercontent.com/121258975/227343982-a43b9ddc-fcdf-4068-82fb-cd57f385a85c.png)
![0](https://user-images.githubusercontent.com/121258975/227342838-b71bec92-b639-4a56-aba0-83cf27ef5eb2.png)
![01](https://user-images.githubusercontent.com/121258975/227342880-631c8e1b-66eb-4400-9757-135a92c3b1a2.png)

this and a thorough test, where I was able to go further by removing certain files.

![2](https://user-images.githubusercontent.com/121258975/227344214-78ac159a-f6f4-40aa-b56c-c5eee4f896d8.png)
![3](https://user-images.githubusercontent.com/121258975/227344297-a0299354-15f1-4d46-9bb2-2bdacd6a2cc4.png)
![4](https://user-images.githubusercontent.com/121258975/227344402-d4b4a456-0d7f-4c97-8946-88138099e8c0.png)
![6](https://user-images.githubusercontent.com/121258975/227344461-488a0285-af4d-4ac1-b493-afe557205d3b.png)

**Comment by Ordinary205** [comment]
Retest

**Comment by dev-channe** [comment]
> Retest

no change or progression.

**Comment by Ordinary205** [comment]
> > Retest
> 
> no change or progression.

So its basically this function that got halted during the game.
![Function](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/d8f27695-80c1-47ef-8229-19040ba65f8b)

**Comment by dev-channe** [comment]
> > > Retest
> > 
> > 
> > no change or progression.
> 
> So its basically this function that got halted during the game. ![Function](https://private-user-images.githubusercontent.com/125112932/302070343-d8f27695-80c1-47ef-8229-19040ba65f8b.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcwMDk5NTMsIm5iZiI6MTcwNzAwOTY1MywicGF0aCI6Ii8xMjUxMTI5MzIvMzAyMDcwMzQzLWQ4ZjI3Njk1LTgwYzEtNDdlZi04MjI5LTE5MDQwYmE2NWY4Yi5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwNFQwMTIwNTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNDAwMGJmYjYwNGQ3MDQwZGUyNzdiMzljYTMyMjI3YzhiZjVkMzY0YmEwMzI0ODA5NGMyMTA0MzljZmE3ZjJkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.y5YsJUhMoFp_g__6S63ID6g6YZMr0y7Vg8AeiqQYv7w)

yes indeed, and also that I did not pay attention to the error in the console which is not the same as the previous one thank you.

**Comment by Ordinary205** [comment]
Retest

**Comment by Hermiten** [comment]
Without deleting some file like they did, it not boot at all.
Without MEMORY hack:

> vkAllocateMemory:VK_ERROR_INVALID_EXTERNAL_HANDLE
> An unhandled exception occurred at $00000001000C4814:
> EAssertionFailed: Assertion failed (vMemory.pas, line 684)
>   $00000001000C4814
>   $000000010009A609
>   $000000010014527B
>   $000000010014578F
>   $000000010013BCE0
>   $00000001000917C3
>   $0000000100094CA0
>   $0000000100090D0C
>   $0000000100005176
>   $0000000100014611
>   $00007FFFD639257D
>   $00007FFFD758AA58
> 
> XInput quit!
> An unhandled exception occurred at $000000010023D06B:
> EAccessViolation: Access violation
>   $000000010023D06B
>   $000000010023F04B
>   $00007FFFD71C8241
>   $00007FFFD71C7D01
>   $00000001002402CD
>   $0000000100183C00
>   $00000001001840F1
>   $00000001001A51B5
>   $000000010018408B
>   $000000010006D748
>   $0000000100004891
>   $0000000100005176
>   $0000000100017F10
>   $0000000100003340
>   $00007FFFD639257D
>   $00007FFFD758AA58

And with it:

> An unhandled exception occurred at $00007FFF3CBF003F:
> EAccessViolation: Access violation
>   $00007FFF3CBF003F
>   $00007FFF3CC36590
>   $00007FFF3CC0FF21
>   $00007FFF3CB0C543
>   $00007FFF3CB23BBD
>   $00007FFFB5EB2577
>   $0000000100131FF4
>   $0000000100131E8F
>   $00000001000942B0
>   $00000001000949E5
>   $0000000100094E60
>   $0000000100090D0C
>   $0000000100005176
>   $0000000100014611
>   $00007FFFD639257D
>   $00007FFFD758AA58
> 
> XInput quit!
> An unhandled exception occurred at $000000010023D06B:
> EAccessViolation: Access violation
